modules/developer-cli-odo-installing-odo-on-linux.adoc

```diff
@@ -6,88 +6,40 @@
 
 = Installing {odo-title} on Linux
 
-== Binary installation
+The `{odo-title}` CLI is available to download as a binary and as a tarball for multiple operating systems and architectures including: 
 
-.Procedure
-
-. Obtain the binary:
-+
-[source,terminal]
-----
-# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o /usr/local/bin/odo
-----
-
-. Change the permissions on the file:
-+
-[source,terminal]
-----
-# chmod +x /usr/local/bin/odo
-----
+[cols="2,1,1",options="header"]
+|===
+|Operating System|Binary|Tarball
+|Linux|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64[odo-linux-amd64] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz[odo-linux-amd64.tar.gz]
+|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[odo-linux-ppc64le] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[odo-linux-ppc64le.tar.gz]
+|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[odo-linux-s390x] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[odo-linux-s390x.tar.gz]
+|===
 
-== Tarball installation
 
 .Procedure
 
-. Obtain the tarball:
+. Navigate to link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[] and download the appropriate file for your operating system and architecture.
+. If you downloaded a tarball, extract the binary:
```

Comment:
So, are steps 2-5 all reliant on the user downloading a tarball instead of a different odo file?

If so, then this whole procedure should be focused on users downloading a tarball. In which case, it would go something like this:

1. Navigate to the page `link:<link>[odo latest], and download the file, `tarball.` 
2. Extract the binary:
3. Change the permissions on the binary: 
So on and so forth. 

If users can download whatever file they want, and only step 2 is contingent on them installing a tarball, then step 2 is optional (because they only do this one step if it they downloaded a tarball). So that would look something link:

1. Navigate to the page, `link:<link>[odo latest]`, and download the appropriate file for your operating system and architecture.

- If you downloaded a tarball, extract the binary:

2. Change the permissions on the binary. 
So on and so forth. 

Honestly, reading the procedure, I think the whole thing is expecting users to download the tarball file, so I would simply instruct them to do so in step one.

Reply:
Agreed. I have reworked the section to add table to downloads, clarify binary vs tarball differences

Agreed about only using the tarball option, but I inherited these docs and that is the pattern used. I will propose removing the binary download documentation in a subsequent JIRA, but there may be good reasons for keeping them...